components/src/main/java/uk/gov/hmrc/components/organism/summary/SummaryRowView.kt

```diff
@@ -125,6 +134,21 @@ class SummaryRowView @JvmOverloads constructor(
         }
     }
 
+    fun setChevronIcon(@DrawableRes icon: Int) {
+        if (icon != NO_CHEVRON_ICON) {
+            binding.imageChevron.setImageResource(icon)
+        }
+    }
+
+    fun setChevronTintColor(@ColorRes iconTint: Int) {
```

Comment:
I think these could be renamed for clarity, as it's not always restricted to a chevron anymore.
```suggestion
    fun setIcon(@DrawableRes icon: Int) {
        if (icon != NO_CHEVRON_ICON) {
            binding.imageChevron.setImageResource(icon)
        }
    }

    fun setIconTintColor(@ColorRes iconTint: Int) {
```